src/xdocs/checks/coding/multiplestringliterals.xml

```diff
@@ -78,7 +78,7 @@ public class MyClass {
   public void myTest() {
     String a2 = &quot;StringContents&quot;; // violation, &quot;StringContents&quot; occurs twice
     String a3 = &quot;DoubleString&quot; + &quot;DoubleString&quot;; // violation, &quot;DoubleString&quot; occurs twice
-    String a4 = &quot;SingleString&quot;; // OK
```

Comment:
@555vedant ,
Sorry but you still don't seem to understand the requirement of the issue.

The suppression you have removed is related  to the java input file, so please find the respective path.